en/lessons/ecto/basics.md

```diff
@@ -71,7 +71,8 @@ config :friends, Friends.Repo,
   hostname: "localhost"
 ```
 
-This configures how Ecto will connect to the database.
+This configures how Ecto will connect to the database. Either edit the username to match one of your existing database roles, or create a new one in your `psql` shell as follows: 
```

Comment:
Let's just note that you may need to configure the connection details to match your particular database's needs.

I'm afraid if we start going down the rabbit hole of SQL commands it won't just stop with this one, we'll want to add MySQL, MSSQL, etc.

Reply:
ok, sure